packages/amplify-java-runtime-provider/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { FunctionRuntimeContributorFactory } from 'amplify-function-plugin-interface';
+import { buildResource } from './utils/build';
+import { packageResource } from './utils/Package';
+export const functionRuntimeContributorFactory: FunctionRuntimeContributorFactory = context => {
+  context.amplify;
+  return {
+     contribute: selection => {
+       if (selection !== 'java') {
+         return Promise.reject(new Error(`Unknown selection ${selection}`));
+       }
+       return Promise.resolve({
+         runtime: {
+           name: 'Java',
+           value: 'java',
+           cloudTemplateValue: 'java8',
+           defaultHandler: 'example.HelloPojo',
+         },
+      });
+    },
+    checkDependencies: () => Promise.resolve({ hasRequiredDependencies: true }),
```

Comment:
Probably want to actually make sure java and javac are on the path

Reply:
done